PhysicsTools/NanoAOD/python/protons_cff.py

```diff
@@ -0,0 +1,73 @@
+                             tagTrackLite         = cms.InputTag("ctppsLocalTrackLiteProducer")
+)
+
+singleRPTable = cms.EDProducer("SimpleProtonTrackFlatTableProducer",
+    src = cms.InputTag("ctppsProtons","singleRP"),
+    cut = cms.string(""),
+    name = cms.string("Proton_singleRP"),
+    doc  = cms.string("bon"),
+    singleton = cms.bool(False),
+    extension = cms.bool(False),
+    skipNonExistingSrc = cms.bool(True),
+    variables = cms.PSet(
+        xi = Var("xi",float,doc="xi or dp/p",precision=12),
+        xiUnc = Var("xiError",float,doc="uncertainty on xi or dp/p",precision=8),
+        thetaY = Var("thetaY",float,doc="th y",precision=13),
+        thetaYUnc = Var("thetaYError",float,doc="theta Y uncertainty",precision=8),
+    ),
+    externalVariables = cms.PSet(
+        decRPId = ExtVar("protonTable:protonRPId",int,doc="Detector ID",precision=8), 
+        sector45 = ExtVar("protonTable:singleRPsector45",bool,doc="LHC sector 45"),
+        sector56 = ExtVar("protonTable:singleRPsector56",bool,doc="LHC sector 56"),
+    ),
+)
+
+multiRPTable = cms.EDProducer("SimpleProtonTrackFlatTableProducer",
+    src = cms.InputTag("ctppsProtons","multiRP"),
+    cut = cms.string(""),
+    name = cms.string("Proton_multiRP"),
+    doc  = cms.string("bon"),
+    singleton = cms.bool(False),
+    extension = cms.bool(False),
+    skipNonExistingSrc = cms.bool(True),
+    variables = cms.PSet(
+        xi = Var("xi",float,doc="xi or dp/p",precision=12),
+        xiUnc = Var("xiError",float,doc="uncertainty on xi or dp/p",precision=8),
+        vtxY = Var("vy()",float,doc="vertex y",precision=11),
+        vtxYUnc = Var("vyError",float,doc="vy Error",precision=8),
+        pt = Var("pt",float,doc="pt",precision=15),
+        thetaX = Var("thetaX",float,doc="theta x",precision=13),
+        thetaXUnc = Var("thetaXError",float,doc="theta X uncertainty",precision=8),
+        thetaY = Var("thetaY",float,doc="theta y",precision=13),
+        thetaYUnc = Var("thetaYError",float,doc="theta Y uncertainty",precision=8),
+        chi2 = Var("chi2",float,doc="chi 2",precision=15),
+        ndof = Var("ndof()",int, doc="n dof", precision=8),
+        t = Var("t",float,doc="t",precision=13),
+        validFit = Var("validFit",bool,doc="valid Fit"),
+        time = Var("time()",float,doc="time",precision=16),
+        timeUnc = Var("timeError",float,doc="time uncertainty",precision=13),
+    ),
+    externalVariables = cms.PSet(
+        sector45 = ExtVar("protonTable:multiRPsector45",bool,doc="LHC sector 45"),
+        sector56 = ExtVar("protonTable:multiRPsector56",bool,doc="LHC sector 56"),
+    ),
+)
+
+protonTables = cms.Sequence(    
+    protonTable
+    +singleRPTable
+    +multiRPTable
+)
+
+for modifier in run2_miniAOD_80XLegacy, run2_nanoAOD_94XMiniAODv1, run2_nanoAOD_94XMiniAODv2, run2_nanoAOD_94X2016:
+    modifier.toReplaceWith(protonTables, cms.Sequence())
```

Comment:
Add the run2_nanoAOD_102Xv1 here so that EOY is completely skipped

Reply:
Thanks for your guidance @mariadalfonso !

Reply:
Done

Reply:
@jwill24
this is actually not done now (or if was done 24 days ago, not got modified) .

this cause crush in our integration testing for miniAOD data of type run2_nanoAOD_102Xv1

Reply:
This has been fixed in the most recent commit